src/main/java/seedu/nutrigoals/logic/commands/ProfileCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.nutrigoals.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.nutrigoals.logic.commands.exceptions.CommandException;
+import seedu.nutrigoals.model.Model;
+
+/**
+ * Displays the user's details
+ */
+public class ProfileCommand extends Command {
+
+    public static final String COMMAND_WORD = "profile";
+    public static final String MESSAGE_SUCCESS = "Profile created: \n %s";
```

Comment:
Just to clarify, `profile` displays the user's profile while `setup` is the command that creates a user profile right? In that case, would it be better to have 'Listed profile' as the success message instead of 'Profile created:'?